src/v2/Apps/Auction/Components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -74,6 +74,11 @@ const AuctionDetails: React.FC<AuctionDetailsProps> = ({ sale, me }) => {
           )}
         </Flex>
       </Flex>
+      <Flex alignItems="center" justifyContent="space-between">
+        <Text variant="md" pr={2}>
+          {sale.auctionsDetailCascadingIntervalLabel}
```

Comment:
Does it really make sense to generate this text in Metaphysics, when it's very simple and just refers to the time? Or are we doing this in case we change the units to minutes?

Reply:
We'll need it for Eigen, so I think it's nice to have the logic somewhere central. There's also [a little more logic](https://github.com/artsy/metaphysics/blob/02246518290d75ee3a6aa92585aebe119b279ba4/src/schema/v2/sale/index.ts#L251) than just showing the string so I'm a fan of keeping it in MP

Reply:
yeah that's what I was going to say as well, there is logic for determining when to show it in metaphysics